src/mscorlib/model.xml

```diff
@@ -1086,6 +1086,9 @@
       <Member Name="StartProfileOptimization(System.String)" />
       <Member MemberType="Event" Name="Resolving" />
       <Member MemberType="Event" Name="Unloading" />
+      <Member MemberType="Event" Name="AssemblyLoad" />
```

Comment:
nit: keep sorted, that can help avoid merge conflicts and is useful when diffing the file against a generated one.